zetaclient/evm/evm_signer.go

```diff
@@ -324,246 +332,233 @@ func (signer *Signer) SignCommandTx(
 		return signedTX, nil
 	}
 
-	return nil, fmt.Errorf("SignCommandTx: unknown command %s", cmd)
+	return nil, fmt.Errorf("SignCommandTx: unknown command %s", txData.cmd)
 }
 
-func (signer *Signer) TryProcessOutTx(
-	cctx *types.CrossChainTx,
-	outTxMan *outtxprocessor.Processor,
-	outTxID string,
-	chainclient interfaces.ChainClient,
-	zetaBridge interfaces.ZetaCoreBridger,
-	height uint64,
-) {
-	logger := signer.logger.Std.With().
-		Str("outTxID", outTxID).
-		Str("SendHash", cctx.Index).
-		Logger()
-	logger.Info().Msgf("start processing outTxID %s", outTxID)
-	logger.Info().Msgf("EVM Chain TryProcessOutTx: %s, value %d to %s", cctx.Index, cctx.GetCurrentOutTxParam().Amount.BigInt(), cctx.GetCurrentOutTxParam().Receiver)
-
-	defer func() {
-		outTxMan.EndTryProcess(outTxID)
-	}()
-	myID := zetaBridge.GetKeys().GetOperatorAddress()
-
-	var to ethcommon.Address
-	var toChain *common.Chain
+func SetChainAndSender(cctx *types.CrossChainTx, logger zerolog.Logger, txData *TransactionData) bool {
```

Comment:
I think we can use `txData` as a receiver for the method to make things clearer
```
func (txData *TransactionData) SetChainAndSender(cctx *types.CrossChainTx, logger zerolog.Logger)
```
It is also unclear what the bool represent